### ISSUE TITLE
Connect to remote notebook

### ISSUE THREAD

**Issue by maximsch2** [jupyter/atom-notebook#34]
It doesn't look like there is a way to connect to a remote notebook server, right? 

Are there plans to add it? Any pointers on where I should look to add it?

**Comment by gnestor** [comment]
See https://github.com/jupyter/atom-notebook/blob/master/lib/main.js#L25 where there is an opener for .ipynb files. I'm not sure about the particulars, but you could download the remote file and open the local file as is, you could access the remote file via FTP (see https://atom.io/packages/remote-edit), or you access the remote file via SSH (https://atom.io/packages/remote-atom).

**Comment by TyberiusPrime** [comment]
@gnestor I don't think reading the .ipynb is what maximsch2 had in mind - I certainly don't.
I have a notebook running on a beefy server somewhere, or even a jupyterhub, and I'd like to access it within atom.

**Comment by gnestor** [comment]
@TyberiusPrime You're right, I read that wrong. There is currently no support for remote notebook servers, but I welcome PRs to add that! 

I have held off on adding any new features to this project until nteract's component are consumable outside of Electron. You can track the progress of that effort here: https://github.com/nteract/nteract/issues/1346

nteract doesn't support remote kernels yet, but it's on the roadmap. You can track the progress of that effort here: https://github.com/nteract/nteract/issues/822
